internal/sourcemap/elasticsearch.go

```diff
@@ -58,11 +59,17 @@ type esSourcemapResponse struct {
 		}
```

Comment:
Any struct tags needed for Total and Value?

Reply:
Added :+1: